install-shared.bash

```diff
@@ -62,15 +60,8 @@ build_from_source() {
         exit 1
     fi
 
-    pushd $__rust_sources_path
-
+    pushd "${__rust_sources_path}" || exit
     cargo --version
-
-    if [[ -f "./scripts/build-release.sh" ]]; then
-        ./scripts/build-release.sh $__library_name $(cat rust-toolchain)
-    else
-        cargo build --release --all
-    fi
-
-    popd
+    cargo build --release --all
```

Comment:
Uh oh! We need to run the `build-release.sh` script in order to get the [proper linker flags to pack into the pkg-config file](https://github.com/filecoin-project/filecoin-ffi/blob/master/rust/scripts/build-release.sh#L31). 

Given that there's only one FFI lib now, artifacts like `install-shared.bash` and the `build-release.sh` can probably be collapsed into a single file (to make things a little more-obvious for maintainers).